### PR TITLE
New color set and token names, plus TOC skeleton

### PR DESCRIPTION
**Pull request by MichelleElliott**
Supports newest color token set defined by designers.
Supports toggle between old and new color token set by changing code slightly.
Adds skeleton of TOC to the app.

Ready for refactor into separate frameworks.  Refactoring will include adding public visibility and Obj-C support where needed.

### CODE REVIEW

File: OTKit-iOS/Design Showcase/Colors/ColorView.swift [hunk 1]
```diff
@@ -11,25 +11,25 @@ struct ColorView: View {
...
+//
+//        Group {
+//            Text("Danger (Dark Mode)")
+//                .frame(width: 300, height: 25)
+//                .foregroundColor(ColorTuple.danger.foregroundColor)
+//                .border(ColorTuple.danger.borderColor, width: 2)
+//                .environment(\.colorScheme, .dark)
+//        }
+//        .background(ColorTuple.danger.backgroundColor)
+//        .environment(\.colorScheme, .dark)
```
Comment (jagdeep-manik) [inline OTKit-iOS/Design Showcase/Colors/ColorView.swift:32]: did we want to keep this?

Comment (MichelleElliott) [reply OTKit-iOS/Design Showcase/Colors/ColorView.swift:32]: I want to have something like that in the code to show how different colors go together as a group, but it was a distraction and not well designed yet.  I will delete it and redo it way better when I can.

File: OTKit-iOS/Token Layer/Colors/ColorToken.swift [hunk 2]
```diff
@@ -11,12 +11,20 @@ import SwiftUI
 enum ColorToken: String, CaseIterable  {
     
     func color() -> Color {
```
Comment (randeep-ot) [inline OTKit-iOS/Token Layer/Colors/ColorToken.swift:13]: Can we simplify this class?
Suggest adding UIColorToken for UIKit and ColorToken could be for SwiftUI

**Comment by MichelleElliott** [comment]
Closing in favor of an approach that makes for cleaner code at usage site.

File: OTKit-iOS/UIKit/Color+ColorToken.swift [hunk 3]
```diff
@@ -0,0 +1,159 @@
...
+    static let foregroundOnSuccess: Color = ColorToken.foregroundOnSuccess.color()
+
+    static let foregroundOnWarning: Color = ColorToken.foregroundOnWarning.color()
+
+    static let foregroundSuccess: Color = ColorToken.foregroundSuccess.color()
+    
+    static let foregroundWarning: Color = ColorToken.foregroundWarning.color()
+
+    
+    static let  backgroundAccentAqua: Color = ColorToken.backgroundAccentAqua.color()
```
Comment (jagdeep-manik) [inline OTKit-iOS/UIKit/Color+ColorToken.swift:96]: extra space between let and background

**Review by jagdeep-manik — APPROVED** [review]
Looking good! Left minor clean-up comments but I'm good to approve.

**Review by jagdeep-manik — APPROVED** [review]
awesome, ty!!

**Comment by MichelleElliott** [comment]
I'm going to merge this with just the one approval so that I can move the code into GC, where it will go through another round of PR reviews.

**Pull request merged**